aws-lc-sys/builder/cmake_builder.rs

```diff
@@ -148,6 +171,24 @@ impl CmakeBuilder {
             }
         }
 
+        if target_os() == "android" {
+            cmake_cfg.define("CMAKE_SYSTEM_NAME", "Android");
+
+            let target = target();
+            let proc = target.split('-').find(|_| true).unwrap();
```

Comment:
I think you can just call `.next()`?

```suggestion
            let proc = target.split('-').next().unwrap();
```